frontend/src/app/pages/landing-page/landing-page/landing-page.component.html

```diff
@@ -80,4 +79,76 @@ <h1>Digitale FüSim MANV</h1>
+            2021 und 2022. Sie stellt eine digitale Umsetzung der
+            Führungssimulation MANV (FüSim MANV) dar, einer von der
+            <a
+                href="https://www.bbk.bund.de/DE/Themen/Akademie-BABZ/akademie-babz_node.html"
+                >Bundesakademie für Bevölkerungsschutz und Zivile
+                Verteidigung</a
+            >
+            am
+            <a href="https://www.bbk.bund.de/"
+                >Bundesamt für Bevölkerungsschutz und Katastrophenhilfe</a
+            >
+            entwickelten Simulation für medizinische Führungskräfte für den Fall
+            eines Massenabfalls an Verletzten (MANV).
+        </p>
+        <p>
+            Der Quelltext dieser Webseite und der dahinterstehenden Technologien
+            steht als Open-Source-Software unter
+            <a href="https://github.com/hpi-sam/digital-fuesim-manv"
+                >https://github.com/hpi-sam/digital-fuesim-manv</a
+            >
+            bereit. Weitere Informationen über das Projekt finden Sie auf der
+            <a
+                href="https://hpi.de/teaching/bachelor-projects/digitales-fuehrungssimulationstraining.html"
+                >Projektwebseite</a
+            >.
+        </p>
+        <h4>Projektbeteiligte</h4>
+        <div class="p-3">
+            <a href="https://www.hpi.de/">
+                <img
+                    class="me-5 m-3"
+                    style="height: 100px"
+                    src="https://www13.hpi.uni-potsdam.de/fileadmin/user_upload/fachgebiete/giese/Bachelorprojekte/2021/hpi-logo.jpg"
+                    alt="HPI"
+            /></a>
+            <a href="https://www.bbk.bund.de/">
+                <img
+                    class="me-5 m-3"
+                    style="height: 100px"
+                    src="https://www13.hpi.uni-potsdam.de/fileadmin/user_upload/fachgebiete/giese/Bachelorprojekte/2021/bbk-logo.jpg"
+                    alt="BBK"
+            /></a>
+            <a href="https://www.malteser-berlin.de/">
+                <img
+                    class="m-3"
+                    style="height: 100px"
+                    src="https://www13.hpi.uni-potsdam.de/fileadmin/user_upload/fachgebiete/giese/Bachelorprojekte/2021/malteser-logo.jpg"
+                    alt="Malteser Berlin"
+            /></a>
+        </div>
+        <div>
+            <span class="fw-bold">Projektteam:</span>
+            <a href="https://github.com/Dassderdie" class="ms-1"
+                >Julian Schmidt</a
+            >, <a href="https://github.com/ClFeSc">Clemens Schielicke</a>,
+            Florian Krummrey, Marvin Müller-Mettnau,
+            <a href="https://hpi.de/giese/people/christian-zoellner.html"
+                >Christian Zöllner</a
+            >
+            und
+            <a href="https://hpi.de/giese/people/matthias-barkowsky.html"
+                >Matthias Barkowsky</a
+            >
```

Comment:
please put me last. in the scientific context, the supervisor / owner always goes last :-)